src/sql/util.py

```diff
@@ -250,56 +250,89 @@ def parse_toml_error(e, file_path):
         return e
 
 
-def get_user_configs(file_path):
+def get_user_configs(primary_path, alternate_path):
     """
     Returns saved configuration settings in a toml file from given file_path
 
     Parameters
     ----------
-    file_path : str
+    primary_path : Path
+        file path to a toml file
+    alternate_path : Path
         file path to a toml file
 
     Returns
     -------
     dict
         saved configuration settings
+    Path
+        the path of the file used to get user configurations
     """
-    data = load_toml(file_path)
-    section_names = ["tool", "jupysql", "SqlMagic"]
-    while section_names:
-        section_to_find, sections_from_user = section_names.pop(0), data.keys()
-        if section_to_find not in sections_from_user:
-            close_match = difflib.get_close_matches(section_to_find, sections_from_user)
-            if not close_match:
-                MESSAGE_PREFIX = (
-                    f"Tip: You may define configurations in "
-                    f"{file_path}. Please review our "
+    data = None
+    tip_displayed = False
+    disable_tip = False
+    for file_path in [primary_path, alternate_path]:
+        if file_path is None or not file_path.exists():
+            continue
+
+        if file_path == alternate_path:
+            STATUS = "Did not find pyproject.toml"
```

Comment:
this can be in lowercase

Reply:
Also please move this piece of code for deciding the final message ina. separate internal function with proper docstring

Reply:
@neelasha23 I did that and the function for deciding the final message turned out to be more complex than the current implementation as the messages are decided on the current file being looked at and each file can have multiple states and some of these states are interdependent. I modified the code and tried to make the message generation as simple as possible so it should be much easier to follow now.